src/pages/configeditor/overview.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Overview"
+excerpt: "Overview."
+categories: configeditor
+slug: overview
+toc: true
+---
+<!-- ---
+
+copyright:
+  years: 2022
+lastupdated: "2022-09-13"
+
+keywords: annotator clinical data, clinical data, annotation
+
+subcollection: wh-acd
```

Comment:
What is a subcollection?

Reply:
sub-collection is a tag for ibm clouds docs navigation / categorization

Reply:
Since wh-acd was our crn id in IBM so I don't think it's relevant here any more, but as above, maybe doesn't hurt to leave as is?